src/api/route-services/samples.js

```diff
@@ -52,6 +51,31 @@ class SamplesService {
 
     throw Error('Sample not found');
   }
+
+  async updateSamples(projectUuid, body) {
+    const marshalledData = convertToDynamoDbRecord({
+      ':samples': body.samples,
+      ':projectUuid': projectUuid,
+    });
+
+    // Update samples
+    const params = {
+      TableName: this.tableName,
+      Key: {
+        experimentId: { S: body.experimentId },
```

Comment:
It might make more sense to reuse the dynamodb marshall and unmarshall functions for this instead of manually formatting the input yourself